releasenotes/notes/remove-interval-envvar-88c126a791a448a0.yaml

```diff
@@ -0,0 +1,4 @@
+---
+other:
+  - |
+    This change removes the deprecated environment variable `DEFAULT_RUNTIME_METRICS_INTERVAL`.
```

Comment:
```suggestion
    This change removes the deprecated environment variable ``DEFAULT_RUNTIME_METRICS_INTERVAL``.
```

Reply:
I don't think this makes a difference in how the release notes get rendered on github